hooks/src/index.js

```diff
@@ -26,7 +26,7 @@ options._render = vnode => {
 	currentComponent = vnode._component;
 	currentIndex = 0;
 
-	if (currentComponent.__hooks) {
+	if (currentComponent !== undefined && currentComponent.__hooks) {
```

Comment:
You don't need to compare with undefined.
```
if (currentComponent && currentComponent.__hooks)
```

Reply:
Changed